demo-projects/meetup/site/pages/signup.js

```diff
@@ -12,19 +13,25 @@ import { Button, Field, Label, Input } from '../primitives/forms';
 import Navbar from '../components/Navbar';
 import Footer from '../components/Footer';
 import { gridSize, colors } from '../theme';
+import { CREATE_USER } from '../graphql/users';
 
 const { publicRuntimeConfig } = getConfig();
 
 export default () => {
+  const [name, setName] = useState('');
   const [email, setEmail] = useState('');
   const [password, setPassword] = useState('');
   const [isLoading, setIsLoading] = useState(false);
   const [errorState, setErrorState] = useState(false);
   const { isAuthenticated, signin } = useAuth();
   const { meetup } = publicRuntimeConfig;
 
-  const handleSubmit = async event => {
+  const handleSubmit = createUser => {
     event.preventDefault();
```

Comment:
This event is no longer being passed to the function, and refers to the global event. I'm pretty sure that is open to problems.

Reply:
Pretty sure this will grab `window.event`, but better to be explicit:

```jsx
  const handleSubmit = createUser => event => {...}
  
  <form
    onSubmit={handleSubmit(createUser)}
  />
```